roles/setup_http_store/tasks/main.yml

```
@@ -6,7 +6,15 @@
         name: podman
         state: present
       become: true
-
+    
+    - name: Get all environment variables
+      ansible.builtin.shell: printenv
+      register: env_vars
+    
+    - name: Display environment variables
+      ansible.builtin.debug:
+        msg: env_vars.stdout_lines 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)